### PR TITLE
Update/use theme color for completed task strikethrough

### LINKED ISSUE #1000: Free trial: Change strikethrough color in tasklist
Currently, the strikethrough colour is black on a pink text and distracting. Investigate & fix if this is possible to be changed.

### Acceptance criteria

1. Change the strikethrough color to follow text theme color
2. If possible, this change should be propogated to core as a follow-up

### PR DESCRIPTION
**Pull request by moon0326**
### All Submissions:

-   [X] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [X] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes https://github.com/Automattic/wc-calypso-bridge/issues/1000

This PR uses the theme color for completed tasks strikethrough.

### How to test the changes in this Pull Request:

1. Checkout this branch and build the assets.
2. Navigate to `WooCommerce -> Home`
3. Complete a task.
4. Confirm the strikethrough color is the same as currently activated theme color.

### Other information:

-   [X] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ ] Have you written new tests for your changes, as applicable?
-   [ ] Have you created a changelog file for each project being changed, ie `pnpm --filter=<project> changelog add`?
-   [ ] Have you included testing instructions?

### FOR PR REVIEWER ONLY:

-   [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

**Review by ilyasfoo — APPROVED** [review]
Works well, LGTM! 🚢 🚢

**Review by adrianduffell — APPROVED** [review]
Works great! Thanks @moon0326 🚀

File: plugins/woocommerce-admin/client/tasks/tasks.scss [hunk 1]
```diff
@@ -239,8 +239,10 @@
...
 
 .woocommerce-task-list__setup {
-	.woocommerce-experimental-list .woocommerce-experimental-list__item.complete {
+	.woocommerce-experimental-list
+	.woocommerce-experimental-list__item.complete {
 		text-decoration: line-through;
+		color: var(--wp-admin-theme-color);
 
 		.woocommerce-task-list__item-title {
 			color: var(--wp-admin-theme-color);
```
Comment (adrianduffell) [inline plugins/woocommerce-admin/client/tasks/tasks.scss:248]: Nitpick: I think the `color` rule in `.woocommerce-task-list__item-title` is redundant now it's in the parent and can be removed

Comment (moon0326) [reply plugins/woocommerce-admin/client/tasks/tasks.scss:248]: Thank you for the suggestion! I tested it after removing `color`, but it did not work as expected. There are other places where we define color for `.woocommerce-task-list__item-title` selector. We need to explicitly define it here to make sure the theme color applies.

**Pull request merged**